.github/workflows/combine-prs.yml

```diff
@@ -52,20 +53,33 @@ jobs:
               if (pull['user']['login'].match(/^(dependabot\[bot\]|dependabot-preview\[bot\])$/)) {
                 console.log('Branch matched: ' + branch);
                 statusOK = true;
+                // if the PR is not mergeable, bail early.
+                const pull_request = await github.rest.pulls.get({
+                  owner: context.repo.owner,
+                  repo: context.repo.repo,
+                  pull_number: pull['number']
+                });
+                if(pull_request.data.mergeable != true) {
```

Comment:
Does this only check for merge conflicts, or will a branch that is out-of-date also be "not mergeable"?

Reply:
After you asked this, I took some more time to create the conditions necessary, so I set up branch protection on a fork, and confirmed that needing to "Update branch" due to branch protection will **not** mark a branch as unmergeable, so we're good to go!